src/zfs-auto-snapshot.sh

```diff
@@ -209,7 +211,7 @@ do_snapshots () # properties, flags, snapname, oldglob, [targets...]
 
 GETOPT=$(getopt \
   --longoptions=default-exclude,dry-run,fast,skip-scrub,recursive \
-  --longoptions=event:,keep:,label:,prefix:,sep: \
+  --longoptions=event:,keep:,label:,prefix:,local-tz:,sep: \
```

Comment:
The `:` after `local-tz` needs to be removed because this option doesn't take an argument.  I'd also suggest moving it down in to the next line.